src/libraries/System.Formats.Asn1/src/System/Formats/Asn1/AsnReaderOptions.cs

```diff
@@ -0,0 +1,72 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Formats.Asn1
+{
+    /// <summary>
+    ///   Specifies options that modify the behavior of an <see cref="AsnReader"/>.
+    /// </summary>
+    public struct AsnReaderOptions
+    {
+        private const int YearMaxMask = 0xFFFF;
+        private const int DefaultTwoDigitMax = 2049;
+        private const int SkipSortMask = unchecked((int)0x80000000);
+
+        private int _value;
```

Comment:
Why do all of the bit twiddling rather than just defining a ushort field and a bool field?

Reply:
"Instant legacy"? :smile:  It used to have more data all crammed into the one int, though it could always have been a ushort / bool / byte, I suppose.

Cleaning it up :)